pandas/io/parsers.py

```diff
@@ -316,7 +314,9 @@
     be used automatically. In addition, separators longer than 1 character and
     different from ``'\s+'`` will be interpreted as regular expressions and
     will also force the use of the Python parsing engine. Note that regex
-    delimiters are prone to ignoring quoted data. Regex example: ``'\r\t'``"""
+    delimiters are prone to ignoring quoted data. Regex example: ``'\r\t'``
+delimiter : str, default ``None``
+    Alternative argument name for sep."""
 
```

Comment:
I'm not sure why this was originally moved out of `_sep_doc`, however I've put it back as this particular documentation of the `delimiter` parameter only applies to `read_csv` and `read_table`, both of which use `_sep_doc` in their documentation.